src/gateway_gatt_service.erl

```diff
@@ -96,16 +96,17 @@ handle_info({remove, wifi, Network, _Char}=Msg, State=#state{}) ->
     %% re-attempt the remove.
     case connman:start_agent() of
         ok ->
-            case connman:remove(wifi, Network) of
+            Result = connman:remove(wifi, Network),
```

Comment:
I'm tempted to insert a sleep here so connman has some time to settle before I pull the rug out from under it.

Reply:
Meh, lets see it it matters?